src/models/Expense.model.js

```diff
@@ -1,9 +1,47 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
   // your code goes here
+  'Expence',
```

Comment:
There is a typo in the model name 'Expence'. It should be corrected to 'Expense' to ensure consistency and avoid confusion.